rts/lra/lra-annotations/src/main/java/org/jboss/narayana/rts/lra/annotation/LRA.java

```diff
@@ -77,8 +77,7 @@
         REQUIRES_NEW,
 
         /**
-         *  <p>If called outside a transaction context, the method call will return with a 412 Precondition Failed
-         *  HTTP status code</p>
+         *  <p>If called outside a transaction context, the method call will return with a git</p>
```

Comment:
What's a got ;)

Reply:
ou, crap... thank you for spotting this :)